src/main/java/uk/gov/hmcts/reform/sscs/model/hmc/ListingCaseStatus.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.sscs.model.hmc;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum ListingCaseStatus {
+    CASE_CREATED("Case Created"),
+    AWAITING_LISTING("Awaiting Listing"),
+    LISTED("Listed"),
+    PENDING_RELISTING("Pending Relisting"),
+    HEARING_COMPLETED("Hearing Completed"),
+    CASE_CLOSED("Case Closed");
+
+    private final String laCaseStatusLabel;
```

Comment:
Should we change this to listingCaseStatusLabel - also is this field really needed?

Reply:
yes, I assume we need the field, since the mapping only provide a enum value like "Case Closed", it cannot be a enum with space.
rename to listingCaseStatusLabel, will do